Gemfile

```diff
@@ -49,7 +49,8 @@ gem 'awesome_spawn',        '>= 1.4.1'
 gem 'default_value_for'
 gem 'haml_lint',            '~> 0.20.0', :require => false
 gem 'more_core_extensions', '~> 2.0.0',  :require => 'more_core_extensions/all'
-gem 'rubocop',              '~> 0.52.0', :require => false
+gem 'rubocop',              '~> 0.69.0', :require => false
+gem 'rubocop-performance',  '~> 1.3',    :require => false
```

Comment:
What is this gem?

Reply:
Courtesy of Martin P:

```
Post-install message from rubocop:
Performance cops have been removed from RuboCop 0.68. Use the rubocop-performance gem instead.

Put this in your Gemfile.

gem 'rubocop-performance'
And then execute:

$ bundle install
Put this into your .rubocop.yml.

require: rubocop-performance
More information: https://github.com/rubocop-hq/rubocop-perfor
```

Reply:
I'll see if I can get Martin to update the rubocop file on his end as well.

Reply:
Ah thanks!  I noticed that rspec org also has split out Rails and Rspec gems as well...seems they are going towards the pluggable approach.